easyrest/views/restaurant_controler.py

```diff
@@ -0,0 +1,126 @@
+"""This module describe menu controler
+This module describes behavior of /restaurant/{id} and
+/restaurant routes
+"""
+
+from pyramid.view import view_config
+from pyramid.response import Response
+from pyramid.httpexceptions import HTTPNotFound
+
+from ..scripts.json_helpers import wrap
+from ..models.restaurant import Restaurant
+
+
+def asign_tags(rests):
+    """Function for assigning all tags related for each restaurant
+    into restaurant dictionary for later conversion to json
+    Args:
+        rests (list): list of restaurants models returned by the query
+    Returns:
+        List of dictionares in format:
+            [
+                {
+                    [<restaurant key:value>, ]
+                    "tags": [<tag dictionary>, ]
+                },
+            ]"""
```

Comment:
PEP257: Closing quote (`"""`) should be on separate line.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves